docs/developers/setup-documentation.rst

```diff
@@ -18,13 +18,17 @@ Setting up
 This presumes that you're familiar with the command line, and are using OSX or another UNIX-like system.
```

Comment:
A separate page for "Intro to the Command line" in the docs sounds like a good idea.